src/Binovo/ElkarBackupBundle/Resources/views/Default/restorebackup.html.twig

```diff
@@ -0,0 +1,35 @@
+{##
+ # @copyright 2012,2013 Binovo it Human Project, S.L.
+ # @license http://www.opensource.org/licenses/bsd-license.php New-BSD
+ #}
+{% extends 'BinovoElkarBackupBundle:Default:base.html.twig' %}
+{% trans_default_domain 'BinovoElkarBackup' %}
+
+{% block body %}
+
+<form class="form-horizontal" action="{{ path('runRestoreJobBackup', {idClient: idClient, idJob: idJob, path: path, idBackupLocation: idBackupLocation}) }}" method="post" {{ form_enctype(form) }}>
+    <h3>{% trans %}Restore backup process{% endtrans %}</h3>
```

Comment:
It's a bit strange to use process here, maybe job or task?

Reply:
ok